redwood/src/decryption.rs

```diff
@@ -50,6 +51,7 @@ impl<'a> DecryptionHelper for Helper<'a> {
             .keys()
             .secret()
             .with_policy(self.policy, None)
+            .for_storage_encryption()
```

Comment:
You may also want to insert `.supported()` here to filter out keys that are not supported by currently running backend, similarly to: https://gitlab.com/sequoia-pgp/sequoia/blob/main/openpgp/examples/decrypt-with.rs#L72